mail_activity_board/views/mail_activity_view.xml

```diff
@@ -108,27 +89,51 @@
                 <field name="activity_category"/>
                 <templates>
                     <t t-name="kanban-box">
-                        <div t-attf-class="#{kanban_color(record.state)} oe_kanban_content oe_kanban_global_click">
-                            <div class="o_kanban_record_bottom">
-                                <div class="oe_kanban_bottom_left">
-                                    <span t-attf-class="fa #{record.icon}" />
-                                    <span t-attf-class="o_activity_color_#{state}"><field name="activity_category" /> </span>
-                                </div>
-                                <div class="oe_kanban_bottom_right">
-                                    <em>due on: </em><span t-attf-class="o_activity_color_#{state}"><field name="date_deadline" /> </span>
-                                    <strong>
-                                        <t t-esc="activity_type_id" />
+                        <div t-attf-class="oe_kanban_content oe_kanban_global_click">
+                            <div class="oe_kanban_content">
+                                <div>
+                                    <strong class="o_kanban_record_subtitle">
+                                        <span t-attf-class="fa #{record.icon.raw_value}" />
+                                        <field name="summary"/>
                                     </strong>
                                 </div>
+                                <div>
+                                    <strong class="o_kanban_record_title"><field name="res_name"/></strong>
+                                </div>
+
+                                <div class="o_kanban_record_bottom">
+                                    <div class="oe_kanban_bottom_left">
+                                        <t t-if="record.date_deadline.raw_value and record.date_deadline.raw_value lt (new Date())">
```

Comment:
`lt`?

Reply:
If record.date_deadline 
and 
its value is previous previous to today
add the class "text-danger".

Is it badly used?

Reply:
There's still this. `lt` is nothing. Did you mean to use `&lt;`? Also, I think you're comparing a `String` with a `Date`, which will very likely produce false positives (not sure, just guessing...)

Reply:
@Yajo I think it is ready.
The field 'date_deadline' in the class 'mail.activity' is required, so I have removed the check if it is empty.